cdap-common/src/main/java/co/cask/cdap/common/metadata/AbstractMetadataClient.java

```diff
@@ -55,18 +48,15 @@
  * Common implementation of methods to interact with metadata service over HTTP.
  */
 public abstract class AbstractMetadataClient {
-  private static final Type SET_METADATA_RECORD_TYPE = new TypeToken<Set<MetadataRecord>>() { }.getType();
-  private static final Type SET_METADATA_RECORD_V2_TYPE = new TypeToken<Set<MetadataRecordV2>>() { }.getType();
+  private static final Type SET_METADATA_RECORD_V2_TYPE = new TypeToken<Set<MetadataRecord>>() { }.getType();
```

Comment:
This can be renamed to SET_METADATA_RECORD_TYPE

Reply:
good point